stanford/roles

```diff
@@ -1 +0,0 @@
-../playbooks/roles
```

Comment:
We used to symlink this to edX's default `roles` directory;
now we have our own (in addition to, not in place of).